src/components/SearchForm/SearchForm.test.js

```diff
@@ -0,0 +1,52 @@
+// Test that component renders an input with
+// the value equal to initial value passed in props
+// Test that after typing to the input and a "click" event on the Submit button,
+// the "onChange" prop is called with proper value
+// Test that after typing to the input and pressing Enter key,
+// the "onChange" prop is called with proper value
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { SearchForm } from './SearchForm';
+
+describe('SearchForm', () => {
+    it('renders the initial value provided in props', () => {
+        const initialQuery = 'query';
+        render(<SearchForm
+                initialQuery={initialQuery}
+                onSearch={() => {}}/>
+        );
+
+        const inputValue = screen.getByPlaceholderText(/What do you want to watch?/i);
+        expect(inputValue).toHaveValue(initialQuery);
+    })
+
+    it('calls the onChange prop with the input value after a "click" event on the Submit button', () => {
+        const onChange = jest.fn();
+        render(<SearchForm
+            initialQuery=''
+            onSearch={onChange}/>
+        );
+
+        const input = screen.getByPlaceholderText(/What do you want to watch?/i);
```

Comment:
good practice is to use `data-testid` attribute and `screen.getByTestId` selector

Reply:
Thanks. Will add this update in Task 4 :)